src/TLNetwork.ts

```diff
@@ -80,7 +80,7 @@ export class TLNetwork {
   /**
    * @hidden
    */
-  public signer: TxSigner
+  public signer: TLWallet
```

Comment:
Not sure if this should still be called signer or change it to wallet?

Reply:
After our discussion now, this should probably stay signer.